pkg/metrics/metrics.go

```diff
@@ -4,11 +4,15 @@
 package metrics
 
 import (
+	"fmt"
+	"golang.org/x/exp/slices"
 	"net/http"
 	"sync"
 	"time"
```

Comment:
```suggestion
	"net/http"
	"sync"
	"time"

    "golang.org/x/exp/slices"
```
The lint message:
```
File is not `goimports`-ed (goimports)
```
is not very clear, but what it means is that the imports are not in the expected order. `"golang.org/x/exp/slices"` should come after the standard library and before the other imports.